l10n_ch_adr_report/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+move to oca
```

Comment:
@sonhd91 Does it have a convention? Or it's just a random text?

Reply:
It just there from begining, should I remove it ?

Reply:
Yes, so that file can be removed